lib/index.js

```diff
@@ -83,13 +85,47 @@ function getFooterNode() {
  *
  */
 
+function getItemValue(item) {
+  return item.place_name
+}
+
+function render(item) {
+  var placeName = item.place_name.split(',');
+  return '<div class="mapboxgl-ctrl-geocoder--suggestion"><div class="mapboxgl-ctrl-geocoder--suggestion-title">' + placeName[0]+ '</div><div class="mapboxgl-ctrl-geocoder--suggestion-address">' + placeName.splice(1, placeName.length).join(',') + '</div></div>';
+}
+
+function getItemValueV6(item) {
+  const { name, place_formatted } = item.properties;
+  return name + (place_formatted ? `, ${place_formatted}` : "");
+}
+
+function renderV6(item) {
+  const { name, place_formatted } = item.properties;
+  return `<div class="mapboxgl-ctrl-geocoder--suggestion">
+      ${
+  name
+    ? `<div class="mapboxgl-ctrl-geocoder--suggestion-title">${name}</div>`
+    : ""
+}
+      ${
+  place_formatted
+    ? `<div class="mapboxgl-ctrl-geocoder--suggestion-address">${place_formatted}</div>`
+    : ""
+}
+    </div>`;
+}
```

Comment:
For organizational and IDE purposes, let's move these functions after `function MapboxGeocoder(options) {...}` and before `MapboxGeocoder.prototypes = {...}`. The main reason is that, in my IDE (VS Code), the JSDoc string for the MapboxGeocoder class is showing up on the highlight tooltip for the first function after the doc string, which in this case makes it look like `getItemValue` is a class instead of a function:
![image](https://user-images.githubusercontent.com/39810086/214124550-f15fbdc6-c65c-49b5-a9e1-c41256b2774f.png)

Reply:
nice catch, but it should be moved before MapboxGeocoder class, just also before the JSDoc annotation, cause eslint will give you error if you use something before it's declaration